src/material-experimental/mdc-tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -114,6 +115,12 @@ export class MatTabNav extends _MatTabNavBase implements AfterContentInit {
     this._inkBar = new MatInkBar(this._items);
     super.ngAfterContentInit();
   }
+
+  override ngAfterViewInit() {
+    if (!this.tabPanel && (typeof ngDevMode === 'undefined' || ngDevMode)) {
```

Comment:
nit: this won't catch the case where it starts off with a panel and it gets removed afterwards. Not sure if we need to worry about it.

Reply:
I think that's an exceptional case that we don't need to guard against, but I'm happy to move the check to either a setter or `ngOnChanges` if we feel otherwise.